mkdocs.yml

```
@@ -0,0 +1,45 @@
+site_name: NYAML Docs
+site_description:
+  The documentation for the NYAML tool which is a converter tool to transcode NeXus schema from YAML to XML (.nxdl.xml).
+site_author: "The FAIRmat authors"
+nav:
+  - Home: index.md
+  - Tutorial: tutorials.md
+  - How-To-Guide : how_to_guide.md
+  - Reference: references.md
+  - Explanation: explanations.md
```

```suggestion
  - Explanation: explanations.md
  - Reference: references.md
```
swapping the order to have them in the same way as they are showing up on the main page